operations/node_management/Adding_a_Liquid-cooled_blade_to_a_System.md

```diff
@@ -282,8 +282,38 @@ This procedure will add a liquid-cooled blades from a HPE Cray EX system.
 
 14. Use boot orchestration to power on and boot the nodes. Specify the appropriate BOS template for the node type.
```

Comment:
Typically, we try not to have more than 3 levels of subordination when creating steps. Some of the other problems indicated in my comments below are related to the spacing complications of having so many substeps. I would recommend changing the format for this section to something similar to the following:

---
### Power on and boot the nodes

Use boot orchestration to power on and boot the nodes. Specify the appropriate BOS template for the node type.

14. Determine how the BOS Session template references compute hosts.
   
    Typically, they are referenced by their "Compute" role. However, if they are referenced by xname, then these new nodes should added to the BOS Session template.
    
    ```bash
    ncn-m001# BOS_TEMPLATE=cos-2.0.30-slurm-healthy-compute
    ncn-m001# cray bos sessiontemplate describe $BOS_TEMPLATE --format json|jq '.boot_sets[] | select(.node_list)'
    ```
    
    If this query returns empty, then skip to sub-step 2.
    
    If this query returns with data, then one or more boot sets within the BOS Session template reference nodes explicity by xname. Consider adding new nodes to this list (sub-step 1) or adding them on the command line (sub-step 2).
    
    1. Add new nodes to the list.
        
        1. Dump the current Session template.
            
            ```bash
            ncn-m001# cray bos sessiontemplate describe $BOS_TEMPLATE --format json > tmp.txt
            ```	  

        2. Edit the tmp.txt file adding the new nodes to the node_list.
            
            ```bash
            ncn-m001# vi tmp.txt
            ```
        
    2. Create the Session template.
        
        1. The name of the Session template is determined by the name provided to the `--name` option on the command line. Use the current value of $BOS_TEMPLATE if you want to overwrite the existing Session template. If you want to use the current value, skip this step. Otherwise, provide a different name for the `--name` option. The name specified in tmp.txt is overridden by the value provided by the `--name` option.
        
            ```bash
            ncn-m001# $BOS_TEMPLATE=<New Session Template name>
            ```
            
            ```bash	  
            ncn-m001# cray bos sessiontemplate create --file tmp.txt --name $BOS_TEMPLATE
            ```	  	  
        
        2. Verify that the Session template contains the additional nodes and the proper name.
            
            ```bash	  
            ncn-m001# cray bos sessiontemplate describe $BOS_TEMPLATE --format json
            ```	  	  
	  
15. Boot the nodes.
    
    ```bash
    ncn-m001# cray bos session create --template-uuid $BOS_TEMPLATE \
      --operation reboot --limit x1005c3s0b0n0,x1005c3s0b0n1,x1005c3s0b1n0,x1005c3s0b1n1
    ```